### ISSUE TITLE
Timeline: Screen Reader Does Not Announce Position of TimelineItem in SAP UI5 Webcomponents

### ISSUE THREAD

**Issue by samanr** [UI5/webcomponents#12510]
### Bug Description

The TimelineItem component in the SAP UI5 Webcomponents library does not provide positional announcements when navigating through items using screen readers (e.g., "1 of 3", "2 of 3"). This feature should be available out of the box to ensure accessibility for users relying on screen readers.

Actual Result: The screen reader does not announce the position of TimelineItems. It only reads the item content without positional information, hindering navigation and accessibility for visually impaired users.

### Affected Component

Timeline

### Expected Behaviour

The screen reader should announce the position of each TimelineItem, such as "Item 1 of 3," "Item 2 of 3," etc., as the user navigates through the items.

### Steps to Reproduce

1. Implement a Timeline component containing multiple TimelineItems.
2. Open the application in a browser and activate the screen reader. (JAWS or VoiceOver)
3. Navigate through the TimelineItems using keyboard navigation.
4. Observe the screen reader output.

### Priority

Medium

### UI5 Web Components Version

2.13.1

### Browser

Chrome

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by PetyaMarkovaBogdanova** [comment]
Hi @samanr ,
I tested this sample: https://ui5.github.io/webcomponents/play/#eNrFVG1P2zAQ_itHPgHCSZoOgUJAE2VISEOgFTZN6hfXvrYGx44St1Ch_ved87KVlyK2L1MiJfec7-65F99ToIzEx3Dmch2kT4HhOQbpOrgXCGscGkdwtsUYFJovp6WdG8kmVktg7GRksq2zq8HNz-sv4I084L-guZkejwI0o4Aw_2Qz5JL-gd6sckuNXthNjXXbqcSJMih34IkwqSofKgVjDR4RsPL2tf9aP7GlQMmE1bZkXN7NK_fsLGRR598LOToOYsbLCh1Rur05Z4c1q1blMyd8ofChsKUbBdDmTeCDkm52LHGhBLJa2ANllFNcs0pwjce9MG6dOeUo5pDnhcYsaiRKO2rzpt-xlUuoqZHrMRf3bTXrTFJY8HKbsYoXp79VA6_ZOYKCS6nMNIWkxPyoDfhGTxiati_-wFztM6dy1FTbk5cAUw5zqGkyh48-W0pIo6QCVPPxM0USh3ESJnHvAHq9tE8Zg6IikaaYUd1JbIs4dNyoSvMFh1OunV3wVseEVuKej6kmWfSKxv8jt4HNB_j8QLzXSxgujQAGg2s4w0pNzVsED8L4sCG4TQzjmM73Evru_KHqZ8lIXrat9bE1H6M-ufwGw6vzOIEk_NRvyDaKes7_kft3JdHCwJoJlmgEwoCKS6xuL_bf4N-nGe95_ofEP2n59zfxf7faxO3KeFKQIzqa6I1ZvIA3DXwz7LRQRKkKB25Z-Ni5lXPtO1-VwotcmfCuqrvdHKwvpr-OzcWMut317pUKVnudq7WN2SHr6zLaXfcyo2rDbjQyKvcLBkZBGK2pq3nh4ZogbbDXtjWDZ_afqTTRA46FJcRQzCqipemir34yOkfvnmcTZUvVWN20Ne4M_9Lsghr2sZh-Vlqm9dX8SMA1m27GGrNgtfoFB9dPrg
with Voiceover and the reading seems fine. Can you, please, provide a video of the test that you execute in order to not hear the reading of the position of timeline items?
Thank you!
Best regards,
Petya Markova.
UI5 Web Components Dispatcher

**Comment by samanr** [comment]
@PetyaMarkovaBogdanova  Thanks for testing. This issue has been on Webcomponents-React Timeline component. I was suggested to create a ticket on this repository. Is there a way we can move it to the webcomponents-react repository and test there?

**Comment by PetyaMarkovaBogdanova** [comment]
@Lukas742 , perhaps you can help us move the issue to web components for react issues?
Thank you!
Kind regards,
Petya Markova.

**Comment by Lukas742** [comment]
Hi @PetyaMarkovaBogdanova , @samanr 

except for the `ExpandableText` component, where we added some props for legacy support (in v1 we offered an `ExpandableText` component in our project), ui5 web components are only wrapped by our project. The underlying markup, attributes, properties, etc. are not touched by that. That's why the accessibility behavior is the same as when using web components directly. 

I checked this behavior with VoiceOver and for me the position announcement are made as well. Although, some screen readers infer these announcements while others don't. When looking at the implementation I can't find the respective [aria attributes](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Attributes/aria-posinset) on the `listitem` element:

```
<div data-sap-focus-ref="true" class="ui5-tli-bubble" role="listitem" tabindex="0" aria-label="Stanislava Baltova, called, 20.02.2017 11:30">
```

So, to be sure I asked @samanr to open an issue here so the owners can check, as I'm not sure how it works with JAWS or NVDA.

___

@samanr could you please let us know which screen reader and which version of it you're using, so the team can reproduce this issue more easily?

**Comment by samanr** [comment]
I am using VoiceOver on Mac. The Timeline reads how many items are there in the list, For example List 2 items. But when the focus is on one of the two items, it does not mention the position.
I can confirm that it is not read using JAWS either. 
@Lukas742

**Comment by samanr** [comment]
I believe this issue can be closed as I was able to ensure the list item is read. 
I had to use the ctrl + opt + shift + left/right arrow to navigate.

**Comment by dimovpetar** [comment]
Hello @samanr ,

I am closing the issue. In case you have further comments, feel free to reopen it.

Best regards,
Petar
